walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +275,146 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void fullyBookedOptionalAttendee() {
+    // Have the optional attendee be fully booked. We should see three options because the algorithm
+    // ignores them
+    //
+    // Events  :       |--A--|     |--B--|
+    //           |-------------*C--------------|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3",
+            TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TimeRange.END_OF_DAY, true),
```

Comment:
nit: when supplying literals as parameters, it's good to indicate the name of the parameter in a comment. E.g.

`TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TimeRange.END_OF_DAY, /* inclusive = */ true)`